cmd/release.go

```diff
@@ -64,14 +74,29 @@ func uploadTarball(path string, f os.FileInfo, err error) error {
 	}
 
 	if matched {
-		sh("github-release upload",
-			"--user", info.Owner,
-			"--repo", info.Name,
-			"--tag", fmt.Sprintf("v%s", info.Version),
-			"--name", fileName,
-			"--file", path)
+		err := retry.Do(func(attempt int) (bool, error) {
+			var err error
+			err = uploadTarball(fileName, path)
+			if err != nil {
+				time.Sleep(2 * time.Second)
+			}
+			return attempt < allowedRetry+1, err
```

Comment:
Your retry package already does checking of max retries (but against MaxRetries = 10, not the flag value), so the slightly different second check here is a bit odd. Why not just have one limit?